### PR TITLE
Mark failing napari tests as xfail

### PR DESCRIPTION
**Pull request by IgorTatarnikov**
Before submitting a pull request (PR), please read the [contributing guide](https://github.com/brainglobe/.github/blob/main/CONTRIBUTING.md).

Please fill out as much of this template as you can, but if you have any problems or questions, just leave a comment and we will help out :)

## Description

**What is this PR**

- [ ] Bug fix
- [ ] Addition of a new feature
- [x] Other

**Why is this PR needed?**
Tests are failing due to an error in the teardown of the `make_napari_viewer` fixture.

**What does this PR do?**
Marks tests with xfail.

## References
https://github.com/brainglobe/cellfinder/issues/443

## How has this PR been tested?
All tests pass locally in a fresh environment.

## Is this a breaking change?
No

## Does this PR require an update to the documentation?
No

## Checklist:

- [x] The code has been tested locally
- [ ] Tests have been added to cover all new functionality (unit & integration)
- [ ] The documentation has been updated to reflect any changes
- [x] The code has been formatted with [pre-commit](https://pre-commit.com/)

### CODE REVIEW

File: tests/test_brainreg_napari.py [hunk 1]
```diff
@@ -78,6 +78,9 @@ def test_workflow(make_napari_viewer, tmp_path):
     assert boundaries.name == "Boundaries"
 
 
+@pytest.mark.xfail(
+    reason="See https://github.com/brainglobe/cellfinder/issues/443"
```
Comment (alessandrofelder) [inline tests/test_brainreg_napari.py:82]: Would 
```suggestion
    reason="See https://github.com/brainglobe/cellfinder/issues/443",
    raises=AssertionError
```
be a slight improvement? (have checked the syntax locally, and that this doesn't xfail if I put `raises=ValueError` instead`.

File: tests/test_brainreg_napari.py [hunk 2]
```diff
@@ -167,6 +170,9 @@ def check_orientation_output(viewer, brain_layer, atlas):
     assert viewer.layers[3].data.shape == (atlas.shape[0], atlas.shape[1])
 
 
+@pytest.mark.xfail(
+    reason="See https://github.com/brainglobe/cellfinder/issues/443"
```
Comment (alessandrofelder) [inline tests/test_brainreg_napari.py:174]:
```suggestion
    reason="See https://github.com/brainglobe/cellfinder/issues/443",
    raises=AssertionError
```
as above.

**Review by alessandrofelder — APPROVED** [review]
Tiny suggestions.

File: tests/test_brainreg_napari.py (hunk 1 above)
Comment (IgorTatarnikov) [reply tests/test_brainreg_napari.py:82]: Good call, should've just copied it from `cellfinder`!

File: tests/test_brainreg_napari.py (hunk 2 above)
Comment (IgorTatarnikov) [reply tests/test_brainreg_napari.py:174]: Good call, should've just copied it from `cellfinder`!

**Review by alessandrofelder — APPROVED** [review]

**Pull request merged**